src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/ApprenticeshipUpdateController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.CommitmentsV2.Api.Types.Responses;
+using SFA.DAS.CommitmentsV2.Application.Queries.GetApprenticeshipUpdate;
+using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+
+namespace SFA.DAS.CommitmentsV2.Api.Controllers
+{
+    [ApiController]
+    [Authorize]
+    [Route("api/apprenticeships/{ApprenticeshipId}")]
+    public class ApprenticeshipUpdateController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+        private readonly IModelMapper _modelMapper;
+
+        public ApprenticeshipUpdateController(IMediator mediator, IModelMapper modelMapper)
+        {
+            _mediator = mediator;
+            _modelMapper = modelMapper;
+        }
+
+        [HttpGet]
+        [Route("pending-update")]
```

Comment:
...and this route could be `pending` so that the whole thing reads:

`api/apprenticeships/{ApprenticeshipId}/updates/pending`

thoughts? 🤔